start.py

```diff
@@ -1011,7 +1014,13 @@ def sigterm_handler(_signo, _stack_frame):
         m2ee.stop()
         sys.exit(0)
 
+    def sigusr_handler(_signo, _stack_frame):
+        metrics.emit(jvm = {'errors': float(_signo == signal.SIGUSR1),
+                            'ooms': float(_signo == signal.SIGUSR2)})
```

Comment:
When this happens `loop_until_process_dies(...)` will terminate with another `emit(...)`. Confirmed with HansT